pkg/trace/obfuscate/sql_tokenizer.go

```diff
@@ -242,7 +254,7 @@ func (tkn *SQLTokenizer) scanIdentifier() (TokenKind, []byte) {
 	}
 	upper := bytes.ToUpper(buffer.Bytes())
 	if keywordID, found := keywords[string(upper)]; found {
-		return keywordID, upper
+		return keywordID, buffer.Bytes()
```

Comment:
Is there any change to test coverage needed for this? We now retain the original case right?

Reply:
Indeed. I'm not sure why before we weren't. I don't think there are any problems since we already have thorough test coverage but I'll ponder over it and double check.